### DISCUSSION TITLE
CNCF Special Purpose Operating System Working Group Participation

### DISCUSSION THREAD

**Discussion by rajaskakodkar** [kairos-io/kairos#1660] [General]
Hello Kairos maintainers,

As part of [CNCF TAG Runtime](https://github.com/cncf/tag-runtime), I am reaching out to let you know that there has been an interest in the community, which includes folks from Flatcar and Bottlerocket, to collaborate by creating a special purpose operating system working group under the CNCF umbrella. It would be great for you to participate in this working group for defining common standards and best practices across this space.

If you are interested, please join [#wg-sp-os](https://cloud-native.slack.com/archives/C05J0J20FC2) on CNCF slack. Let me know if you have any questions.

Thank you!

**Comment by rajaskakodkar** [comment]
cc: @raravena80 @helayoty @nikhita

**Comment by mudler** [comment]
Hi @rajaskakodkar ! Thank you for reaching out - this is actually something we are preparing recently as well: https://github.com/kairos-io/kairos/issues/1545. We would like to donate Kairos to the CNCF foundation, so we are preparing the needed material, I'll reach out on slack!

Thanks!
